Feature 1174 client intelligent polling (#1260)
mercedes-benz/sechub: sechub-doc/src/docs/asciidoc/documents/code2doc/client/02_sechub_client.adoc

 - approve job as being ready to start
 - wait for job done,
 - fetch automatically result report to output folder (html or json)

>  - returns non-zero on `red` (so usually your build will break on critical or high findings)

Reviewer: ```suggestion
 - returns non-zero on `red` (in most cases your build will break on critical or high findings)
```

Author: changed wording

> IMPORTANT: For security reasons always pass your api token via the environment variable `SECHUB_APITOKEN`.

Reviewer: ```suggestion
IMPORTANT: For security reasons always pass your API-Token via the environment variable `SECHUB_APITOKEN`.
```
